### ISSUE TITLE
Add ability to share the lobste.rs link itself

### ISSUE THREAD

**Issue by ubitux** [twodayslate/claw#54]
It's useful to be able to save/share the link to the comment page, not just the target site. For example the HN app this, allowing to share to personal notes with `[title](https://ycombinator.com/...)`. It's also possible to link/share a specific comment.

**Comment by twodayslate** [comment]
![image](https://github.com/twodayslate/claw/assets/1085707/3f05ae60-a8d2-4f50-95ab-7531b0efd787)

**Comment by ubitux** [comment]
Oh I'm sorry, I actually mistaken this repository for the Claw implementation on Android which seems unrelated.

(For the record, a long press on the comment didn't help on Android, but it allowed me to discover that I could drag the comment to open the comments page in a browser where I could share, so thank you, and sorry for the noise)

**Comment by twodayslate** [comment]
Got it. Thanks I actually didn't know there was an android application. 

https://github.com/msfjarvis/compose-lobsters

**Comment by ubitux** [comment]
> Got it. Thanks I actually didn't know there was an android application.
> 
> https://github.com/msfjarvis/compose-lobsters

Oh, how did you manage to find it? I don't see any direct link from the play store nor from within the app?

**Comment by twodayslate** [comment]
I just googled "claw lobsters github". It actually shows up before this repository for me 🤷‍♂️

![image](https://github.com/twodayslate/claw/assets/1085707/5b430ac9-6092-4b14-8db1-684645ba65e0)

I've contacted the other developer to see if we can eliminate this type of confusion in the future. @msfjarvis

**Comment by msfjarvis** [comment]
> > Got it. Thanks I actually didn't know there was an android application.
> > [msfjarvis/compose-lobsters](https://github.com/msfjarvis/compose-lobsters)
> 
> Oh, how did you manage to find it? I don't see any direct link from the play store nor from within the app?

Sorry about that, I somehow didn't realize that my Play Store listing was missing a source link. As I've informed @twodayslate over email, the Play Store description has been updated with a link to the GitHub repository which should show up on Google Play in a few hours, and I will also add a settings option in the app to make reporting bugs or sending feature requests simpler.

Apologies again to both parties for the mix up.
